osu.Game/Overlays/Rankings/SpotlightSelector.cs

```diff
@@ -88,17 +101,24 @@ private void load(OverlayColourProvider colourProvider)
             background.Colour = colourProvider.Dark3;
         }
 
-        protected override void LoadComplete()
+        public void ShowInfo(APISpotlight spotlight, int mapCount)
```

Comment:
This might as well have accepted the entire `GetSpotlightRankingsResponse` and saved a method argument.

Reply:
Please, explain further, I don't really understand what are you trying to tell, sorry

Reply:
In the overlay you're doing:
```csharp
header.SpotlightSelector.ShowInfo(spotlightRequest.Result.Spotlight, spotlightRequest.Result.BeatmapSets.Count);
```
while you could just do:
```csharp
header.SpotlightSelector.ShowInfo(spotlightRequest.Result);
```
and make `ShowInfo` only accept a single `GetSpotlightRankingsResponse`.

Reply:
Now I got it, good catch